packages/plugin-core/src/test/suite-integ/NoteLookupCommand.test.ts

```diff
@@ -671,5 +713,596 @@ suite("NoteLookupCommand", function () {
         }
       });
     });
+
+    test("horizontal split basic", (done) => {
+      runLegacyMultiWorkspaceTest({
+        ctx,
+        preSetupHook: async ({ wsRoot, vaults }) => {
+          await ENGINE_HOOKS.setupBasic({ wsRoot, vaults });
+        },
+        onInit: async ({ vaults, engine }) => {
+          const cmd = new NoteLookupCommand();
+          stubVaultPick(vaults);
+
+          // close all editors before running.
+          VSCodeUtils.closeAllEditors();
+
+          await VSCodeUtils.openNote(engine.notes["foo"]);
+          await cmd.run({
+            initialValue: "bar",
+            splitType: "horizontal",
+            noConfirm: true,
+          });
+          const barEditor = VSCodeUtils.getActiveTextEditor();
+          expect(barEditor!.viewColumn).toEqual(2);
+
+          await cmd.run({
+            initialValue: "foo.ch1",
+            splitType: "horizontal",
+            noConfirm: true,
+          })
+          const fooChildEditor = VSCodeUtils.getActiveTextEditor();
+          expect(fooChildEditor!.viewColumn).toEqual(3);
+
+          done();
+        }
+      });
+    });
+
+    test("horizontal split modifier toggle", (done) => {
+      runLegacyMultiWorkspaceTest({
+        ctx,
+        preSetupHook: async ({ wsRoot, vaults }) => {
+          await ENGINE_HOOKS.setupBasic({ wsRoot, vaults });
+        },
+        onInit: async ({ vaults }) => {
+          const cmd = new NoteLookupCommand();
+          stubVaultPick(vaults);
+
+          const { controller } = await cmd.gatherInputs({
+            splitType: "horizontal"
+          });
+
+          controller.quickpick.show();
+
+          let { horizontalSplitBtn } = getSplitTypeButtons(
+            controller.quickpick.buttons
+          );
+
+          expect(horizontalSplitBtn?.pressed).toBeTruthy();
+
+          await controller.onTriggerButton(horizontalSplitBtn);
+
+          ({ horizontalSplitBtn } = getSplitTypeButtons(
+            controller.quickpick.buttons
+          ));
+
+          expect(horizontalSplitBtn.pressed).toBeFalsy();
+
+          done();
+        }
+      });
+    });
+
+    test("copyNoteLink basic", (done) => {
```

Comment:
the reason `copyNoteLink basic` test doesn't work is because button effects get triggered on `prepareQuickPick`. at this point, the picker has no items attached to it. that only happens during `showQuickPick`. this doesn't affect other modifiers because they don't rely on the `picker` having items attached to it. 

what I would do in this case is just remove the `CopyNoteLink` tests for now - we might modify it later but right now its not worth the effort

Reply:
I'll revert it back so the test skips (pushed after enabling again last night to demonstrate where it failed).

I won't spend time fixing it further, but wouldn't `controller.quickpick.show()` below attach the items to the quickpick?